internal/pkg/template/workload_test.go

```diff
@@ -265,18 +265,38 @@ func TestHTTPTargetContainer_IsHTTPS(t *testing.T) {
 	require.False(t, HTTPTargetContainer{Port: "8080"}.IsHTTPS())
 }
 
-func TestSsmOrSecretARN_RequiresSub(t *testing.T) {
-	require.False(t, ssmOrSecretARN{}.RequiresSub(), "SSM Parameter Store or secret ARNs do not require !Sub")
+func TestPlainSSMOrSecretARN_RequiresSub(t *testing.T) {
+	require.False(t, plainSSMOrSecretARN{}.RequiresSub(), "SSM Parameter Store or secret ARNs do not require !Sub")
 }
 
-func TestSsmOrSecretARN_ValueFrom(t *testing.T) {
-	require.Equal(t, "/github/token", SecretFromSSMOrARN("/github/token").ValueFrom())
+func TestPlainSSMOrSecretARN_RequiresImport(t *testing.T) {
+	require.False(t, plainSSMOrSecretARN{}.RequiresImport(), "secret name that refers to a value imported from CloudFormation Stack")
```

Comment:
maybe we need a better description of the test case:
```suggestion
	require.False(t, plainSSMOrSecretARN{}.RequiresImport(), "plain SSM Parameter Store or secret ARNs do not require !ImportValue")
```
and similar for the other test cases.